pwa/src/templates/dashboard/Dashboard.tsx

```diff
@@ -100,6 +102,15 @@ const Dashboard: React.FC = () => {
             subtitle="View all endpoints"
             linkTo="endpoints"
           />
+
+          <DashboardCard
+            amount={getRepositoriesQuery.isSuccess ? getRepositoriesQuery.data.length : 0}
+            title="Github Shop"
+            iconBackgroundColor="5bc0de"
+            icon={<i className="fas fa-shopping-cart fa-lg"/>}
+            subtitle="View all github repositories"
+            linkTo="shop"
+          />
```

Comment:
I'm getting the following error on first load:

![image](https://user-images.githubusercontent.com/66552932/160546372-9b406255-88c2-474f-8ce9-ce6e5bf55a34.png)

Reply:
Your `Repository.getAll()` is returning `undefined`. Something in the call itself is failing:

![image](https://user-images.githubusercontent.com/66552932/160547186-df59dfbd-021a-4385-8860-9978205e17f5.png)